fairer-iOS/fairer-iOS/Screens/Home/UIComponent/WeekCalendarView/PickDateView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  PickDateView.swift
+//  fairer-iOS
+//
+//  Created by 홍준혁 on 2023/02/11.
+//
+
+import UIKit
+
+import SnapKit
+
+final class PickDateView: BaseUIView {
+    
+    // MARK: - property
+    
+    var changeClosure: ((Date)->())?
+    var dismissClosure: ((Date,Date,String,String)->())?
+    private lazy var changeDateResult = Date()
+    private lazy var changeDateStartDateWeekResult = Date()
+    private lazy var datePicker: UIDatePicker = {
+        let view = UIDatePicker()
+        view.datePickerMode = .date
+        view.preferredDatePickerStyle = .inline
+        return view
+    }()
+    let blurredEffectView: UIView = {
+        let v = UIView()
+        v.backgroundColor = UIColor.black
+        v.alpha = 0.6
+        return v
+    }()
+    
+    let pickerHolderView: UIView = {
+        let v = UIView()
+        v.backgroundColor = .white
+        v.layer.cornerRadius = 8
+        return v
+    }()
+    
+    // MARK: - life cycle
+    
+    override func configUI() {
+        setAction()
+    }
```

Comment:
제가 생각하기에 setAction은 ui를 만든다는 느낌이 강하지 않아서 configUI 안에 들어가는 것보다는 viewDidLoad에 들어가면 좋을 것 같아요!

override 
```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()
        setAction()
    }
```
은 어떨까요??

Reply:
[36e59b1](https://github.com/fairer-iOS/fairer-iOS/pull/79/commits/36e59b13fe0e770849dc525c34ffde2201d6b14c)